package.json

```diff
@@ -5,18 +5,23 @@
   "scripts": {
     "bootstrap": "yarn install && node ./scripts/bootstrap",
     "lint": "node_modules/.bin/eslint src/ test/ scripts/",
-    "test": "jest --config=test/config.json"
+    "test": "jest --config=test/config.json",
+    "new": "node server.js"
```

Comment:
`yarn start` is more idiomatic for a server script

Reply:
done